omnigibson/simulator.py

```diff
@@ -1236,6 +1236,7 @@ def _init_stage(
                 position=np.array(m.DEFAULT_VIEWER_CAMERA_POS),
                 orientation=np.array(m.DEFAULT_VIEWER_CAMERA_QUAT),
             )
+            self.viewer_visibility = gm.VIEWER_CAMERA_VISIBLE
```

Comment:
This is set up so that we don't load viewer_camera (which could save vram space if we have multiple other robot vision sesnors)